SerialSimulator/Program.cs

```diff
@@ -55,7 +55,7 @@
 {
     FileName = python,
     // Run for 99999 minutes
-    Arguments = $"{scriptPath} 1",
+    Arguments = $"{scriptPath} 99999",
```

Comment:
This was the reason that our simulator kept cancelling 1 min into running, my bad :(